### PR TITLE
Issue #13321: Kill mutation for SuppressWarningHolderCheck-1

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13321: Kill mutation for SuppressWarningHolderCheck-1

----------

# Check :- 
https://checkstyle.org/checks/annotation/suppresswarningsholder.html#SuppressWarningsHolder

----------

# Mutation
https://github.com/checkstyle/checkstyle/blob/0169b0891563cd7e12d39aa42acc6e6b2cb0ee38/config/pitest-suppressions/pitest-misc-suppressions.xml#L93-L127

---------

# Explaination
Test added

### CODE REVIEW

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+com.puppycrawl.tools.checkstyle.checks.SuppressWarningsHolder
+aliasList = (default)
+
+com.puppycrawl.tools.checkstyle.filters.SuppressWarningsFilter
+
+com.puppycrawl.tools.checkstyle.checks.coding.UnusedLocalVariableCheck
+
+*/
+
+//non-compiled with eclipse: The value for annotation attribute must be a constant expression
```
Comment (romani) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java:11]: Can we avoid this ?
Better to not use this conflict of jdk and eclipse compiler

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java (hunk 1 above)
Comment (Kevin222004) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java]: Done I think ci will not fail

Comment (romani) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java]: This comment was for annotation like `@SuppressWarnings({InputSuppressWarningsHolder9.unusedLocalVariableCheck`

Please simplify Input to do `@SuppressWarnings({"brr"})`

Comment (Kevin222004) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java]: No I can't do that this type of test is created to kill 
https://github.com/checkstyle/checkstyle/blob/0831953d82df93e5ebe7f178c2f2b904af8b8f3b/src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java#L521-L526

> @SuppressWarnings({"brr"}) 

if I change input like this then the ast will change and token will be of type 
https://github.com/checkstyle/checkstyle/blob/0831953d82df93e5ebe7f178c2f2b904af8b8f3b/src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java#L516
which is already covered by existing input

Comment (romani) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java]: Ok, in this case we need such comment, please restore it.
And we need to move file to u compiled file folder, example https://github.com/checkstyle/checkstyle/blob/63d82894e18ce12662521ce7d7042c4b214c84be/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/InputSemicolonBetweenImports.java#L12

Please install Eclipse to confirm it is still issue for it.
https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20non-compiled%20with%20eclipse&type=code

Comment (romani) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java]: https://checkstyle.org/eclipse.html

Comment (Kevin222004) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/suppresswarningsholder/InputSuppressWarningsHolder1.java:11]: @romani I have updated pr after successfully verifying mvn in eclipse id

**Review by romani — APPROVED** [review]
Ok to merge .

I will merge after CI passed

**Pull request merged**